### DISCUSSION TITLE
Tried Uploading Sketch to a ESP32-Wroom32 but there is no web Interface

### DISCUSSION THREAD

**Discussion by webdisaster** [realA10001986/Time-Circuits-Display#12] [Q&A] [answered]
Uploaded Sketch to a new ESP32 and can connect to TCD-AP but cannot connect to Web Interface via Browser. Ping to Device is Possible. Webserver does not respond. Tried it wthout any components connected. Do i need External Devices before start (etc SD card...)

**Comment by realA10001986** [comment]
The ESP must be on the TCD control board, otherwise there might be long delays because of failed hardware initialization. No other hardware needs to be attached.

**Reply by realA10001986** [reply]
On second thought, if the access point is visible, you're past the hw init phase. What URL do you try to connect to?

**Reply by webdisaster** [reply]
yes wlan is visible. But the esp is only connected to power since the pcb is not availlable at the moment.

http://192.168.4.1

**Reply by realA10001986** [reply, marked as answer]
It's probably stuck when polling non-existent i2c devices, accessing missing audio devices or something like that.

**Comment by webdisaster** [comment]
That looks possible to me. Try to find desctivation  points till i will get pcb

**Comment by webdisaster** [comment]
Thanks for great job and helping.

**Comment by realA10001986** [comment]
Now I remember: It goes into an infinite "panic" loop if the RTC (real time clock) is not found. The white LED blinks, but since you haven't one connected, you don't see it.
